config/defaults.conf

```diff
@@ -223,11 +223,18 @@ tokenizer = "MosesTokenizer"  // The name of the tokenizer, passed to the Task c
                               // Use "" for edge probing tasks to use the original tokenization,
                               // or one of 'OpenAI.BPE', 'MosesTokenizer', 'bert-uncased-base',
                               // etc.
-word_embs = none  // The type of word embedding layer. Set to none when using ELMo, GPT, or BERT.
-                  // Options: none, scratch (i.e., trained from scratch), glove, fastText.
+
+input_module = "" // The word embedding layer, available values are the following:
+                  // gpt, glove, ELMo, scratch (i.e., trained from scratch), fastText. For BERT, 
+                  // specify the version of BERT. Available values for BERT are: bert-base-uncased, 
+                  // bert-large-cased. For more, see:
+                  //   https://github.com/huggingface/pytorch-pretrained-BERT
+                  //   #loading-google-ais-pre-trained-weigths-and-pytorch-dump
+                  // For ELMo, BERT, and GPT, there are additional config parameters. 
+
 word_embs_file = ${WORD_EMBS_FILE}  // Path to embeddings file.
 d_word = 300  //  Dimension of word embeddings. Not used by ELMo, GPT, or BERT.
-elmo = 0  // If true, load and use ELMo.
+
 elmo_chars_only = 0  // If true, use *only* the char CNN layer of ELMo. If false but elmo is true,
```

Comment:
Even better: Make this another input_module option called elmo-chars-only or elmo-char-layer